### ISSUE TITLE
Accessibility: OSD toolbar focusable when it should be hidden

### ISSUE THREAD

**Issue by jimarthurnilsen** [NationalLibraryOfNorway/ngx-mime#263]
**Describe the bug**
OSD toolbar is focusable in it's hidden state and readable by screenreaders.
**To Reproduce**

1. Go to 'http://localhost:4200/demo'
2. Open dashboard mode
3. Set focus on mime-viewer-header
4. Tab and the OSD will get focus and show tooltips (after fullscreen button)

**Expected behavior**
OSD toolbar should not be able to get focus when in it's hidden state

**Comment by raymondkarstensen** [comment]
Fixed in https://github.com/NationalLibraryOfNorway/ngx-mime/pull/264
